sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -226,27 +279,25 @@
         }
         return getSecret(secretBase.name(), secretBase.version(), context);
     }
+
     /**
      * Get the latest version of the specified secret from the key vault. The get operation is applicable to any secret stored in Azure Key Vault.
      * This operation requires the {@code secrets/get} permission.
      *
      * <p><strong>Code Samples</strong></p>
      * <p>Gets latest version of the secret in the key vault. Subscribes to the call asynchronously and prints out the
      * returned secret details when a response is received.</p>
-     * <pre>
-     * secretAsyncClient.getSecret("secretName").subscribe(secretResponse -&gt;
-     *   System.out.printf("Secret with name %s , value %s \n", secretResponse.value().name(),
-     *   secretResponse.value().value()));
-     * </pre>
+     * {@codesnippet com.azure.keyvault.secrets.secretclient.getSecret#string}
      *
      * @param name The name of the secret.
      * @throws ResourceNotFoundException when a secret with {@code name} doesn't exist in the key vault.
      * @throws HttpRequestException if {@code name} is empty string.
-     * @return A {@link Mono} containing a {@link Response} whose {@link Response#value() value} contains the requested {@link Secret secret}.
+     * @return A {@link Mono} containing the requested {@link Secret secret}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<Secret>> getSecret(String name) {
-        return getSecret(name, "");
+    public Mono<Secret> getSecret(String name) {
+        return getSecretWithResponse(name, "")
+            .flatMap(FluxUtil::toMono);
```

Comment:
Try to be consistent in two ways:
1. Sometimes you use `this.` and sometimes not. When it comes to method calls like this, don't use `this`.
2. Sometimes you wrap to a new line, and sometimes not. Unless you need to wrap or there are multiple chained calls, don't bother - it's nicer on one line.